hw/ip/alert_handler/alert_handler_component.core

```diff
@@ -0,0 +1,47 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:alert_handler_component:0.1"
+description: "Alert Handler component without the CSRs"
+
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:tlul
+      - lowrisc:prim:all
+    files:
+      - rtl/alert_pkg.sv
+      - rtl/alert_handler_reg_wrap.sv
+      - rtl/alert_handler_class.sv
+      - rtl/alert_handler_ping_timer.sv
+      - rtl/alert_handler_esc_timer.sv
+      - rtl/alert_handler_accu.sv
+      - rtl/alert_handler.sv
+    file_type: systemVerilogSource
+
+  files_verilator_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/alert_handler.vlt
+    file_type: vlt
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/alert_handler.waiver
+    file_type: waiver
+
+
+targets:
+  default: &default_target
+    filesets:
+      - tool_verilator  ? (files_verilator_waiver)
+      - tool_ascentlint ? (files_ascentlint_waiver)
+      - files_rtl
```

Comment:
do we still need these lines since alert_handler.core already have

Reply:
Yes, I think so. Will try running lint without these lines to confirm.

Reply:
On second thoughts, I will defer to @msfschaffner's opinion on this.

Reply:
let's leave this there for now. I will crosscheck lint after this change went in.